src/test/aws-fsx-csi-driver/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: aws-fsx-csi-driver
+description: The Amazon FSx for Lustre Container Storage Interface (CSI) Driver implements CSI specification for container orchestrators (CO) to manage lifecycle of Amazon FSx for Lustre filesystems.
+type: application
+version: 0.1.0
+appVersion: latest
```

Comment:
Better to specify the exact version here.

Reply:
yes, i fix this